config/LSSTCam-imSim/apertures.py

```diff
@@ -0,0 +1,38 @@
+# Set up aperture photometry
+
+# This file is part of ap_pipe.
+#
+# Developed for the LSST Data Management System.
```

Comment:
Do we actually want the copyright header in all our config files? Seems excessive for files that might only have a couple lines in them, and I think I've only ever seen it in obs_lsst files.